celery_pubsub/pubsub.py

```diff
@@ -84,6 +92,17 @@ def _topic_to_re(topic: str) -> re.Pattern[str]:
 _pubsub_manager: PubSubManager = PubSubManager()
 
 
+def subscribe_to(topic: str) -> typing.Callable[[typing.Callable[[P], R]], Task[P, R]]:
+    def decorator(func: typing.Callable[[P], R]) -> Task[P, R]:
+        app_name, module_name = func.__module__.split(".", 1)
+        task_name = f"{app_name}.{module_name}.{func.__qualname__}"
+        task_instance = func if isinstance(func, Task) else task(name=task_name)(func)
```

Comment:
How about not using ternary? I know it's more of a style choice, not a hard recommendation.
But I prefer to have something more readable for future humans, even if it adds a couple lines. :)

Also, we can skip computing the name if we're not going to use it.

```suggestion
        if isinstance(func, Task):
            task_instance: Task[P, R] = func
        else:
            app_name, module_name = func.__module__.split(".", 1)
            task_name = f"{app_name}.{module_name}.{func.__qualname__}"
            task_instance = task(name=task_name)(func)
```